book/website/communication/research-objects.md

```diff
@@ -0,0 +1,39 @@
+(cm-ro)=
+# Research Object to capture the Research Life Cycle
+
+## Prerequisites / recommended skill level
+| Prerequisite |  Importance  | Notes        |
+| ---------------- |------------------ |--------------|
+| {ref}`rr-rdm` | Helpful | RDM lifecyle |
+
+## Summary
+Research outcomes encompass publications, data, software, bibliographical material and any other resources (such as experimental workflows, standards) that can be potentially useful for conducting research.
+
+Research Objects (ROs) {cite:ps}`Garciasilva2019-research-objects` allow working open by design and share during the research process and not only the research outputs at the end.
+
+In this chapter, we will introduce ROs, their typologies and which platform and technologies exist to create and publish them.
+
+```{figure} ../figures/research-object.jpg
+---
+name: ro-lifecyle
+alt: This image shows how research objects evolve and grow in content during the collaboration process and how new research objects can be derived from existing ones.
+---
+Research Objects allow working open by design and share during the research process and not only the research outputs at the end. _The Turing Way_ project illustration by Scriberia. Used under a CC-BY 4.0 licence. DOI: [10.5281/zenodo.3332807](https://doi.org/10.5281/zenodo.3332807).
+```
+
+(cm-ro-motivation)=
+## Background & Motivation
+Research objects (ROs) are living resources helping to organise and describe the inputs, materials, and methods used in a scientific experiment and obtained as a result and not only at the end when publishing the research outcomes. In short, ROs can be seen as a single information unit where any research material can be shared with other scientists at discrete milestones of the investigation. 
+
+The entire research lifecycle can be captured, allowing the release and publication of results progressively, keeping track of versioning and change information. ROs facilitate reproducibility and reuse of the scientific methods and results through access to resources, context and metadata, and the possibility of forking existing ROs. As a result, ROs support evidence and support validation of findings claimed in scholarly articles.
```

Comment:
At the end of this paragraph, these results are the effect of reproducibility. The effect of reusability (e.g. prevent researcher reinvent the wheel, save the time of other group by allowing the research material to be reused in different context).

Reply:
@radifar thanks for your comment, we've split the sentence below:

`ROs facilitate reproducibility and reuse of the scientific methods and results through access to resources, context and metadata, and the possibility of forking existing ROs`

to 

`ROs facilitate reproducibility of the scientific methods and results through access to resources, context and metadata, and reuse with the possibility of forking existing ROs.`

please let us if this is ok, we aren't sure about your last comment.